circle.yml

```diff
@@ -4,8 +4,7 @@ machine:
   node:
     version: v5.5.0
   environment:
-    NODE_012_VERSION: v0.12.0
-    NODE_4_VERSION: v4.2.2
+    NODE_4_VERSION: v4.5.0
```

Comment:
Yeah, that's how it's currently used. I think I have it configured to spin up four VMs to test in different versions. Narrowing to two won't be a huge change.